osi_trafficsign.proto

```diff
@@ -579,7 +579,7 @@ message TrafficSign
                 //
                 TYPE_WARNING_ROAD_NARROWING = 10;
```

Comment:
Now WARNING steps in for DANGER.
Where's the difference? Is there any criterion to distinguish warning and danger?
Again, I would propose to remove it.